src/python/nimbusml/internal/entrypoints/timeseriesprocessing_iidchangepointdetector.py

```diff
@@ -0,0 +1,109 @@
+# - Generated by tools/entrypoint_compiler.py: do not edit by hand
```

Comment:
It looks like we didn't generate a public API for these entrypoints that got added.  Maybe we should add that in a separate PR?

Reply:
Yes, adding them to be public facing should be done as a separate task. Currently by default all entrypoints are generated but only the ones specified in manifest_diff.json will have user facing interface / class

---
In reply to: [243395603](https://github.com/Microsoft/NimbusML/pull/55#discussion_r243395603) [](ancestors = 243395603)